Funcky.NewtonsoftJson/JsonSerializerSettingsExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using Newtonsoft.Json;
+
+namespace Funcky.NewtonsoftJson
+{
+    public static class JsonSerializerSettingsExtensions
+    {
+        public static JsonSerializerSettings AddOptionConverter(this JsonSerializerSettings settings)
```

Comment:
I'm not sure about this API, maybe we should make it more open-ended like `AddFunckyConverters`? Or do we want to expose the converter types directly? What do you think @FreeApophis?

Reply:
We're only exposing the extension method as discussed